components/FunFacts.vue

```diff
@@ -5,14 +5,14 @@
       <ul class="grid grid-cols-1 gap-5 lg:grid-cols-3">
         <li class="bg-blue-light p-5 rounded-md lg:p-12">
           <nuxt-img
-            class="h-auto m-auto my-4 rounded-lg w-32 lg:w-44"
+            class="h-auto m-auto my-8 rounded-lg w-32 lg:w-44"
             format="webp"
             height="336"
             src="/img/hero-images/www.png"
             width="336"
             alt="Ship Shape's octopus mascot wearing headphones and holding a tumbler and microphone"
           />
-          <p>
+          <p class="text-center">
```

Comment:
@hannakim91 I don't think these should be `text-center`. These text sections are centered, but the text is not `text-align: center`. We should instead use `flex justify-center` etc when centering sections like this.

Reply:
W/O the `text-center` the text justifies left like this instead of centering, as in the mockup:

<img width="399" alt="Screen Shot 2022-03-22 at 9 49 27 AM" src="https://user-images.githubusercontent.com/62962173/159532430-d3d0f7c2-f1ee-4a25-b78b-493ff44b0bbd.png">

Reply:
The other two seem to be justified to the left though so I can swap just the middle/Ecologi one out if that is better?